### PR TITLE
Added individual test case docstring support and tests

### PR DESCRIPTION
**Pull request by palakpsheth**
Added ability to define __doc__ per each test case similar to __name__ attribute

### CODE REVIEW

File: test/test_functional.py [hunk 1]
```diff
@@ -236,6 +236,41 @@ class Mytest(object):
...
+    d1.__name__ = 'case1'
+    d1.__doc__ = """docstring1"""
+
+    d2 = Myint(2)
+
+    data_hello = data(d1, d2)(hello)
+    setattr(Mytest, 'test_hello', data_hello)
+
+    ddt_mytest = ddt(Mytest)
+    assert_is_not_none(getattr(getattr(ddt_mytest, 'test_hello_1_case1'),
```
Comment (txels) [inline test/test_functional.py]: Why not assert_equals or something more precise? You know the value you should expect...

**Comment by wswld** [comment]
@palakpsheth sorry for the wait, merging for now, but I'm going to clean up it a bit myself later.

File: test/test_functional.py (hunk 1 above)
Comment (wswld) [reply test/test_functional.py]: Fixed

**Comment by wswld** [comment]
Fixed the issues with docs and tests. This is up for the next release.

**Comment by palakpsheth** [comment]
Thanks!

**Pull request merged**